src/WORKSPACE

```diff
@@ -32,8 +32,8 @@ rules_proto_toolchains()
 
 git_repository(
     name = "com_github_nelhage_rules_boost",
-    commit = "a1dd05e7e9178f8aad86e39f3a5b377902eae5b2",
     remote = "https://github.com/nelhage/rules_boost",
+    branch = "master",
```

Comment:
Changed branch to master, since I made an upstream PR (https://github.com/nelhage/rules_boost/pull/137) to include polygon headers.

Reply:
Hmm, we shouldn't be depending on a branch like this, because it means that something external to this repo can change whether or not it can build. Could we depend on the specific commit on master (`9f9fb8b2f0213989247c9d5c0e814a8451d18d7f`) instead?

Reply:
fixed